src/test/scala/za/co/absa/hyperdrive/trigger/models/tables/SearchableTableQueryTest.scala

```diff
@@ -139,8 +139,28 @@ class SearchableTableQueryTest extends FlatSpec with Matchers with BeforeAndAfte
 
 
   "the date-time-range filter" should "find values within the inclusive range" in {
-    val startDate = LocalDateTime.of(2020, 3, 1, 0, 0, 0)
-    val endDate = LocalDateTime.of(2030, 1, 1, 0, 0, 0)
+    val startDate = Option(LocalDateTime.of(2020, 3, 1, 0, 0, 0))
+    val endDate = Option(LocalDateTime.of(2030, 1, 1, 0, 0, 0))
+    val filter = DateTimeRangeFilterAttributes(field = TestSearchableTableFieldNames.localDateTimeField,
+      start = startDate, end = endDate)
+    val searchRequest = TableSearchRequest(
+      dateTimeRangeFilterAttributes = Some(Seq(filter)),
+      sort = None,
+      from = 0,
+      size = 50
+    )
+
+    val result = await(db.run(underTest.search(searchRequest)))
+
+    val expected = Seq(TestSearchableData.t1, TestSearchableData.t3)
+    result.total should be > 0
+    result.total shouldBe expected.size
+    result.items should contain theSameElementsAs expected
+  }
+
+  "the date-time-range filter" should "find values greater or equal to the left bound" in {
```

Comment:
What about values lower or equal to the right bound?

I would add that test, although it's kinda covered by that test below for the int range. Otherwise, please try to extract the common functionality in `applyIntRange` and `applyDateTimeRange`

Reply:
It is covered by the test bellow. Done as you suggested.